storage/db/migrations/20200304101229_add_payments_frequency_to_app_plans_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Phinx\Db\Adapter\MysqlAdapter;
+
+class AddPaymentsFrequencyToAppPlansTable extends Phinx\Migration\AbstractMigration
+{
+    public function change()
+    {
+        $this->table('apps_plans', [
+            'id' => false,
+            'primary_key' => ['id'],
+            'engine' => 'InnoDB',
+            'encoding' => 'utf8mb4',
+            'collation' => 'utf8mb4_unicode_ci',
+            'comment' => '',
+            'row_format' => 'DYNAMIC',
+        ])
+            ->addColumn('payment_frequency', 'integer', [
+                'null' => false,
```

Comment:
@rwhite27 should allow null since it will break production

Reply:
ok